src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier csp = new ColorSupplier();
```

Comment:
do not use abbreviation names
```suggestion
    private ColorSupplier colorSupplier = new ColorSupplier();
```